### ISSUE TITLE
Bootstrap 4 issues

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/SearchWorks#3598]
Details in Slack: https://stanfordlib.slack.com/archives/G018TJ20XGE/p1696342457503039

- [x] focus color
- [x] ~~placeholder text~~ The production site only has a contrast ratio of 2.84:1 here, so it fails WCAG AA.
- [x] more facets modal pagination controls are mashed together with the sort controls:
- [x] i’m not exactly sure what the current bookmark (“select”) checkbox is aligned to, but it’s now top-aligned… i think the original goal was something like “vertically centered with the first line of the title”, maybe?
- [x] we also seemed to have lost some white-space between the checkbox and the select?
- [x] it feels like the search menu bar controls are vertically positioned higher and look a little funny. i think the goal is vertically centered?
- [x] the whole applied facet value has a pointer cursor, but isn’t actionable:
- [x] location-level request button alignment isn’t great on the catalog search page:
- [x] i’m not sure why the find-it button changed on the search page.. maybe it’s fine?
- [x] (and it also changed on the results page)
- [x] we lost value indentation on the result page:
- [x] Librarian view doesn’t look great:
 the header is massive,
 the “last updated” moved into the body (which might be fine, but needs to be horizontally aligned correctly)
 we lost some vertical whitespace between the marc records and the FOLIO data expand/collapse stuff
- [x] Cite modal title font size is also gigantic
- [x] The “Send to” dropdown menu no longer has the expected keyboard controls, and it also lost its little caret (edited) 
- [x] we lost some vertical space between the articles+ pagination and the bottom footer:
- [x] i’m not sure it was exactly right to begin with, but the articles+ select box and the menu bar vertical alignment is now worse:
- [x] “filter by database title” section isn’t good: https://searchworks-stage.stanford.edu/databases
- [x] we lost a little vertical padding for the title in the masthead panels (e.g. database panel or sdr panel): (edited) 
- [x] the “iiif resources” panel layout changed:
- [x] mini-bento facets are now vertically stacked and take up a lot more space:
- [x] bookplates display is all squished: https://searchworks-stage.stanford.edu/view/6311177
- [x] “show text directions” in the “find it” modal isn’t aligned right (and also has a gigantic title): https://searchworks.stanford.edu/view/6311177
- [x] full screen browse page lost some vertical margin after the menu bar: https://searchworks-stage.stanford.edu/browse?start=14464388&view=gallery
- [x] bookmarks page lost vertical padding before the page title:
- [x] bookmarks bulk cite is missing item title headers

**Comment by saseestone** [comment]
One more issue that I saw today. We've lost whitespace in the Exhibits panel
<img width="391" alt="image" src="https://github.com/sul-dlss/SearchWorks/assets/14897546/2818837d-9893-4f2a-b264-c9879ddf2289">

Example url: https://searchworks-folio-dev.stanford.edu/view/13901909

**Comment by jcoyne** [comment]
Thank you @saseestone. I believe that is fixed now.
